frontend/app/services/PayPalService.scala

```diff
@@ -1,36 +1,43 @@
 package services
 
+import actions.AuthRequest
+import com.gu.identity.play.IdMinimalUser
+import com.gu.okhttp.RequestRunners
+import com.gu.paypal.PayPalConfig
+import com.gu.touchpoint.TouchpointBackendConfig.BackendType
 import com.netaporter.uri.Uri.parseQuery
 import com.typesafe.scalalogging.LazyLogging
 import configuration.Config
 import controllers.PayPal.{PayPalBillingDetails, Token}
 import controllers.routes
 import okhttp3.{FormBody, OkHttpClient, Request, Response}
-import play.api.mvc.RequestHeader
+import play.api.mvc.AnyContent
+import utils.TestUsers
 
 object PayPalService extends LazyLogging {
```

Comment:
To fit in with the other services, `PayPalService` shouldn't be a singleton `object` - instead it should be a class like `StripeService`,`ZuoraService`, etc, taking it's backend-specific config in it's constructor. This lets you make it the PayPalService into  a field on 'MemberService' - so that the 'MemberService` uses only services that are configured for the same backend.